shared/src/prompts.rs

```diff
@@ -565,6 +565,12 @@ pub fn ask_endpoint(listen_port: u16) -> Result<Endpoint, Error> {
         .interact()?
     {
         publicip::get_any(Preference::Ipv4)
+    } else if Confirm::with_theme(&*THEME)
+        .wait_for_newline(true)
+        .with_prompt("You do not have a fixed global IP (use the unspecified address)?")
```

Comment:
This doesn't quite read like a question to me, possible re-wording:

```suggestion
        .with_prompt("Use an unspecified IP address? (this can occur if you do not have a fixed global IP)")
```